src/controller/posts/getAllPost.js

```diff
@@ -0,0 +1,13 @@
+const getPost = require('../../database/queries/post/getPost');
+
+const getAllPosts = (req, res, next) => {
+  getPost()
+    .then((data) => res.json({
+      error: false,
+      data: data.rows,
+    }))
```

Comment:
It's a good practice to maintain consistency throughout the API by following the same structure for the response object in each controller.
In some responses, I saw you set the `statusCode` prop but in this controller you ignore it.